internal/webui/public/js/yaml.min.js

```diff
@@ -0,0 +1 @@
e(i<number){res+=str;i++}return res};Utils.getStringFromFile=function(path,callback){var data,fs,j,len1,name,ref,req,xhr;if(callback==null){callback=null}xhr=null;if(typeof window!=="undefined"&&window!==null){if(window.XMLHttpRequest){xhr=new XMLHttpRequest}else if(window.ActiveXObject){ref=["Msxml2.XMLHTTP.6.0","Msxml2.XMLHTTP.3.0","Msxml2.XMLHTTP","Microsoft.XMLHTTP"];for(j=0,len1=ref.length;j<len1;j++){name=ref[j];try{xhr=new ActiveXObject(name)}catch(undefined){}}}}if(xhr!=null){if(callback!=null){xhr.onreadystatechange=function(){if(xhr.readyState===4){if(xhr.status===200||xhr.status===0){return callback(xhr.responseText)}else{return callback(null)}}};xhr.open("GET",path,true);return xhr.send(null)}else{xhr.open("GET",path,false);xhr.send(null);if(xhr.status===200||xhr.status===0){return xhr.responseText}return null}}else{req=require;fs=req("fs");if(callback!=null){return fs.readFile(path,function(err,data){if(err){return callback(null)}else{return callback(String(data))}})}else{data=fs.readFileSync(path);if(data!=null){return String(data)}return null}}};return Utils}();module.exports=Utils},{"./Pattern":8}],11:[function(require,module,exports){var Dumper,Parser,Utils,Yaml;Parser=require("./Parser");Dumper=require("./Dumper");Utils=require("./Utils");Yaml=function(){function Yaml(){}Yaml.parse=function(input,exceptionOnInvalidType,objectDecoder){if(exceptionOnInvalidType==null){exceptionOnInvalidType=false}if(objectDecoder==null){objectDecoder=null}return(new Parser).parse(input,exceptionOnInvalidType,objectDecoder)};Yaml.parseFile=function(path,callback,exceptionOnInvalidType,objectDecoder){var input;if(callback==null){callback=null}if(exceptionOnInvalidType==null){exceptionOnInvalidType=false}if(objectDecoder==null){objectDecoder=null}if(callback!=null){return Utils.getStringFromFile(path,function(_this){return function(input){var result;result=null;if(input!=null){result=_this.parse(input,exceptionOnInvalidType,objectDecoder)}callback(result)}}(this))}else{input=Utils.getStringFromFile(path);if(input!=null){return this.parse(input,exceptionOnInvalidType,objectDecoder)}return null}};Yaml.dump=function(input,inline,indent,exceptionOnInvalidType,objectEncoder){var yaml;if(inline==null){inline=2}if(indent==null){indent=4}if(exceptionOnInvalidType==null){exceptionOnInvalidType=false}if(objectEncoder==null){objectEncoder=null}yaml=new Dumper;yaml.indentation=indent;return yaml.dump(input,inline,0,exceptionOnInvalidType,objectEncoder)};Yaml.stringify=function(input,inline,indent,exceptionOnInvalidType,objectEncoder){return this.dump(input,inline,indent,exceptionOnInvalidType,objectEncoder)};Yaml.load=function(path,callback,exceptionOnInvalidType,objectDecoder){return this.parseFile(path,callback,exceptionOnInvalidType,objectDecoder)};return Yaml}();if(typeof window!=="undefined"&&window!==null){window.YAML=Yaml}if(typeof window==="undefined"||window===null){this.YAML=Yaml}module.exports=Yaml},{"./Dumper":1,"./Parser":7,"./Utils":10}]},{},[11]);
```

Comment:
what is the source of this js files? and how will they be updated in the future? ... wondering if it needs to have some sort of versioning and some documentation as to how to generate/update when needed

Reply:
good point!

Reply:
I've added a comment for now, but for the future we should look into a way to vendor and generate those assets. However I'd rather do in a follow-up PR as might be changing many things on the way